modules/raft/src/integrationTest/java/org/apache/ignite/raft/jraft/core/ItNodeTest.java

```diff
@@ -247,6 +249,10 @@ public void teardown() throws Exception {
 
         TestUtils.assertAllJraftThreadsStopped();
 
+        if (persistentLogStorageFactory != null) {
+            persistentLogStorageFactory.stopAsync();
```

Comment:
Let's also wait for the future to be completed with `assertThat(..., willCompleteSuccessfully())`

Reply:
Agree. Fixed.